snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/datastore/config/SnomedCoreConfiguration.java

```diff
@@ -101,6 +102,12 @@
 	@NotNull
 	private Set<String> reasonerExcludedModuleIds = Collections.emptySet();
 	
+	@NotNull
+	private String namespaceModuleAssigner;
```

Comment:
Set this to a default value (`default`) otherwise we will need an update to hundreds of configuration files 😱.